examples/cache-with-aws/src/main/java/momento/client/example/doc_examples/DocExamplesJavaAPIs.java

```diff
@@ -0,0 +1,47 @@
+package momento.client.example.doc_examples;
+
+import momento.sdk.auth.CredentialProvider;
+import momento.sdk.exceptions.SdkException;
+import software.amazon.awssdk.auth.credentials.DefaultCredentialsProvider;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.secretsmanager.SecretsManagerClient;
+import software.amazon.awssdk.services.secretsmanager.model.GetSecretValueRequest;
+import software.amazon.awssdk.services.secretsmanager.model.GetSecretValueResponse;
+
+public class DocExamplesJavaAPIs {
+
+  public static void example_API_retrieveAuthTokenFromSecretsManager() {
```

Comment:
I will hook this to public dev docs once this PR gets merged

Reply:
> I will hook this to public dev docs once this PR gets merged.
> 
> One side question: can we change the DocuSaurus configs to pull in a local branch of another Git repo while we are testing this?

In your local checkout of public-dev-docs, you can do `cd sdk-git-repos/client-sdk-java && git fetch && git checkout blah`, as a hacky workaround. we can make something better if we need it :)